jenineellis/fantastic4
# First version of browse for wines

This pull request represents an early version of the Browse page.

### What works

- Displays first 25 wines from database
- Basic API to get data from MongoDB has been implemented
- Uses JSX

### Current Limitations

- Uses a temporary UI; not yet implemented Marta's card design
- Filtering dropdowns are displayed at the top, but not functional

### How to test

1. Run `npm install` to install all dependencies
1. Make sure you have the `.env` file in your project root directory. This has been previously shared in the Google Drive folder.
1. Run `npm start` to start the development server. This will automatically open the browser to `localhost:3000`.
1. Click on the *Wines* link at the top to see a list of first 25 wines from DB.
